buildbot/docker-builders/centos8-64/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:8
+RUN yum install -y epel-release
```

Comment:
Probably don't need EPEL on CentOS8, see other comment

Reply:
the builder uses `mock` which we're getting from epel